### DISCUSSION TITLE
How to do vector normalization？

### DISCUSSION THREAD

**Discussion by WLBRich** [milvus-io/milvus#3937] [Q&A and General discussion] [answered]
I want to use the inner product calculation method, how can I achieve the normalization of the vector？
My method is "feature_norm = feature / np.linalg.norm(feature)", is it right to do so？

**Comment by shiyu22** [comment, marked as answer]
Yes, you are right.

Also, you can check the squared sum of the vectors, if it's 1 it's normalized.
